content/kit/batteries/charging.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Battery Charging
+---
+
+The charging cable is cable-tied to the charger and you must not remove this cable. The charging lead is terminated with 4mm bullet connectors, which present a very high risk of shorting the battery if removed from the charger.
+
+## Charging Checklist
+This list must be followed carefully each time a battery is connected to the charger.
+
+1. Plug in the mains adapter and plug it into the battery charger.
+2. Connect the charging lead from the charger to the battery. Ensure you plug the lead into the charger first and the battery second to avoid risk of shorting the battery.
+3. Connect the monitoring lead from the battery to the charger.
+4. Place the battery in the charging bag.
+5. Configure the charger using the menu system as follows. 
+
+### Starting Charging
+
+{{% notice warning %}}
+Do not attempt to charge your battery if any of the cables are lose or frayed. Contact us immediately if you think there's an issue!
+{{% /notice %}}
+
+1. Press `Start/Enter` to select lithium battery charging.
```

Comment:
This should say "LiPo battery charging", as many chargers (including the IMAX B6s we provide to teams) support other types of lithium battery (e.g. Li-ion, LiFe).